service-front/module/Application/src/Controller/AbstractAuthenticatedController.php

```diff
@@ -102,7 +102,9 @@ public function onDispatch(MvcEvent $e)
             return $authenticated;
         }
 
-        $this->getLogger()->info('Request to ' . get_class($this), $this->identity->toArray());
+        $this->getLogger()->info('Request to ' . get_class($this), [
+            'userId' => $this->identity->id(),
+        ]);
```

Comment:
I spotted this was including identity fields which we don't want to log (token, roles, expiry date) so have restricted it down to just the user's ID